### ISSUE TITLE
Update docs on what to use instead of showall

### ISSUE THREAD

**Issue by mauro3** [JuliaCI/BenchmarkTools.jl#96]
`showall` is gone: https://github.com/JuliaLang/julia/pull/22847 How to best look at results now? Note that `repr` and `print` show even less info.

**Comment by devmotion** [comment]
I was wondering the same and noticed that `show(stdout, MIME"text/plain"(), suite)` displays only a limited and non-verbose summary. For a more verbose output one has to provide the `verbose` and `limit` keywords, see https://github.com/JuliaCI/BenchmarkTools.jl/blob/master/src/groups.jl#L253). In my opinion it would be nice if BenchmarkTools would respect common properties of `IOContext` such as `limit` and `compact`.

**Comment by jrevels** [comment]
Yup, BenchmarkTools was originally written before we had the nice `IOContext` interface. It'd be great for BenchmarkTools to be updated to properly respect it.
